views/backend/write.php

```diff
@@ -7,48 +7,54 @@
 <h2>Poster un article</h2>
 
 <?php
-    if(isset($_POST['post'])){
-        $title = htmlspecialchars(trim($_POST['title']));
-        $content = htmlspecialchars(trim($_POST['content']));
-        $posted = isset($_POST['public']) ? "1" : "0";
+   //Fonction permettant de vérifier les données envoyées par l'utilsiateur
+     function form_page_write(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The function form_page_write() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)